modules/ob-creating-a-buildpacks-build.adoc

```diff
@@ -233,7 +231,7 @@ The `BuildRun` resource creates a `TaskRun` resource, which then creates the pod
 $ oc get pods -w
 ----
 +
-_Example output_
+.Example output
```

Comment:
I am assuming that you will replace such lines with sentences in the future as AFAIK we are not supposed to have block titles within procedure steps going forward. Also, I am not familiar with this syntax where the block title line is separated from the block by the plus sign.